frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -355,20 +343,25 @@ function BookInfo:getCoverImage(doc, file, force_orig)
     return cover_bb
 end
 
-function BookInfo:updateBookInfo(file, book_props, metadata_updated_caller_callback, prop_updated)
+function BookInfo:updateBookInfo(file, book_props, prop_updated, prop_value_old)
     if prop_updated == "cover" and self.ui then
         self.ui.doc_settings:getCoverFile(true) -- reset cover file cache
     end
-    self.prop_updated = prop_updated
+    self.prop_updated = {
+        file = file,
+        doc_props = book_props,
+        prop = prop_updated,
+        prop_value_old = prop_value_old,
```

Comment:
I would be more specific in the names of the keys of this table that will be broadcasted - so it's clearer in remote places:
```lua
    self.prop_updated = {
        filepath = file,             -- it is is really the fullpath
        doc_props = book_props,      -- ok, as it's the same thing available as self.ui.doc_props
        metadata_key_updated = prop_updated,
        metadata_value_old = prop_value_old,
        metadata_value_new = book_props[prop_updated],
               -- (if that is the right way to get it) no matter if it's duplicated, it will be clearer for the remote code.
    }
```